components/butterfly/src/server/mod.rs

```diff
@@ -119,6 +120,7 @@ impl Clone for Server {
             gossip_rounds: self.gossip_rounds.clone(),
             blacklist: self.blacklist.clone(),
             socket: None,
+            peer_seed: Arc::clone(&self.peer_seed),
```

Comment:
I don't understand why this is here, rather than using the `member_list.initial_members` array.